components/a11y/demos/dev/10-visuallyHidden.js

```diff
@@ -1,5 +1,3 @@
-/** @jsx jsx */
```

Comment:
I wonder it would be easier if we replace `/** @jsx jsx */` with `/** @jsxImportSource @emotion/react */`

Reply:
This is fine. As per the discussion it's better to manage it via Babel config since it's more robust.